ext/spl/php_spl.stub.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+function iterator_to_array(Traversable $iterator, bool $use_keys = true): array {}
+
+function iterator_count(Traversable $iterator): int {}
+
+function iterator_apply(Traversable $iterator, callable $function, ?array $args = []): int {}
```

Comment:
It seems to me that this should rather be:
```suggestion
function iterator_apply(Traversable $iterator, callable $function, ?array $args = null): ?int {}
```